tool/tctl/main.go

```diff
@@ -32,6 +32,8 @@ func main() {
 		&common.AccessRequestCommand{},
 		&common.AppsCommand{},
 		&common.DBCommand{},
+		&common.KubeCommand{},
```

Comment:
The same needs to be done in Enterprise implementation of tctl in `teleport.e` repo.